kubernetes/resource_kubernetes_persistent_volume_claim_test.go

```diff
@@ -429,12 +427,51 @@ func TestAccKubernetesPersistentVolumeClaim_googleCloud_storageClass(t *testing.
 	})
 }
 
-func TestAccKubernetesPersistentVolumeClaim_expansion(t *testing.T) {
```

Comment:
I broke this test into two: one for minikube and one for GKE. This will clear up the errors we see in TC when the test is skipped. (`ID-only refresh check never ran.`)